app/helpers/helpers/helpers_file.go

```diff
@@ -8,6 +8,39 @@ import (
 	"strings"
 )
 
+func GetCommand(line string) string {
+	return line[2:]
+}
+
+func IsTestFile(text string) bool {
+	return text == "t"
+}
+
+func CreateGolangTestFile(text string) string {
+	return text[:len(text)-3] + "_test.go"
+}
+
+func ConvertToFiles(text, packageName string) []string {
+	var res []string
+	files := strings.Split(text, " ")
```

Comment:
let us create the `res` after files
this way I believe we can set the size

Reply:
errors  ```go res := make([]string, len(files), len(files))```

Reply:
because of this ```go res[ix] = fileTrimmed
			res[ix+1] = str```